sympy/functions/elementary/miscellaneous.py

```diff
@@ -57,10 +57,11 @@ def sqrt(arg):
 
     >>> from sympy import sqrt, Symbol
     >>> x = Symbol('x')
-
+    >>> from sympy import Rational
+    >>> sqrt(2) == 2**Rational(1,2)
+    True
```

Comment:
I think that's exposing implementation detail, namely, that `sqrt(2)` will return `Pow(2,Rational(1,2))`.
The proper way to test whether two SymPy expressions are mathematically equal is via `Eq` I think.
(I'm slightly outside my comfort zone here, so you might want to ask in the mailing list what's the proper way to test for equality.)